src/main/java/org/broadinstitute/hellbender/dev/pipelines/bqsr/BaseRecalibratorDataflowUtils.java

```diff
@@ -90,6 +102,21 @@ public static void ensureReferenceIsReadable(final PipelineOptions popts, String
   // ---------------------------------------------------------------------------------------------------
   // non-public methods
 
+  private static PCollection<BaseRecalOutput> quantize(SAMFileHeader readsHeader, BaseRecalibrationArgumentCollection toolArgs, PCollection<RecalibrationTables> recal) {
+    return recal.apply(ParDo
+            .named("quantize")
```

Comment:
This `ParDo` named "quantize" is not actually doing quantization of the quality scores, right? I assume it's just adding the quantization table to the final output table? If that's correct, you should rename this `ParDo` (and enclosing method) to something less alarming such as "AddQuantizationTable".